src/main/java/org/fundacionjala/coding/melvi/MovieRefactor/Customer.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.melvi.MovieRefactor;
+
+import java.util.Vector;
+
+/**
+ *
+ */
+public class Customer implements ICustomer {
+    private String name;
+    private Vector rentals = new Vector();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "Vector" by an unsynchronized one such as "ArrayList" or "LinkedList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)